plugin/pkg/auth/authorizer/rbac/bootstrappolicy/testdata/controller-roles.yaml

```diff
@@ -1043,9 +1043,10 @@ items:
   - apiGroups:
     - ""
     resources:
+    - services
     - services/status
     verbs:
-    - update
+    - patch
```

Comment:
@kubernetes/sig-auth-api-reviews is this backwards compatible and safe to do?

Reply:
upgrades will preserve the existing permission and add to it, so it is safe... but I would probably include update and patch in the role instead